modules/local/multiqc.nf

```diff
@@ -1,70 +1,69 @@
 process MULTIQC {
     label 'process_medium'
-
-    conda "bioconda::multiqc=1.23"
-    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/multiqc:1.23--pyhdfd78af_0' :
-        'biocontainers/multiqc:1.23--pyhdfd78af_0' }"
+    conda "bioconda::multiqc=1.25.1"
+    container "${workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container
+        ? 'https://depot.galaxyproject.org/singularity/multiqc:1.25.1--pyhdfd78af_0'
+        : 'biocontainers/multiqc:1.25.1--pyhdfd78af_0'}"
 
     input:
     path workflow_summary
     path multiqc_config
     path mqc_custom_config
     path logo
 
-    path ('fastqc/*')
-    path ('trimgalore/fastqc/*')
-    path ('trimgalore/*')
+    path 'fastqc/*'
+    path 'trimgalore/fastqc/*'
+    path 'trimgalore/*'
```

Comment:
Actually, to be more consistent with standard modules implementation I would suggest this instead:
```suggestion
    path "fastqc/*"
    path "trimgalore/fastqc/*"
    path "trimgalore/*"
```